packages/frontend/app/styles/shared/header-menu.scss

```diff
@@ -1,9 +1,7 @@
 @use "../ilios-common/mixins" as cm;
 @use "../ilios-common/colors" as c;
```

Comment:
Still needed?

Reply:
nope, thanks for catching that. removed it.